server/service/integration_mdm_test.go

```diff
@@ -563,8 +563,9 @@ func (s *integrationMDMTestSuite) TestDEPProfileAssignment() {
 	require.NotNil(t, cmd.Command)
 	require.Equal(t, "InstallEnterpriseApplication", cmd.Command.RequestType)
 	require.NotNil(t, cmd.Command.InstallEnterpriseApplication)
-	require.NotNil(t, cmd.Command.InstallEnterpriseApplication.ManifestURL)
-	require.Contains(t, *cmd.Command.InstallEnterpriseApplication.ManifestURL, apple_mdm.FleetdPublicManifestURL)
+	// TODO: this seems flaky?
+	//require.NotNil(t, cmd.Command.InstallEnterpriseApplication.ManifestURL)
+	//require.Contains(t, *cmd.Command.InstallEnterpriseApplication.ManifestURL, apple_mdm.FleetdPublicManifestURL)
```

Comment:
Took a quick look but couldn't figure it out immediately, has it been flaky for anyone else? Happy to re-enable this, but for me it was helpful to comment it out while running tests.

Reply:
Yes, this has been flaky for me as well. @roperzh is planning to take a look today.

Reply:
I think I figured it out https://github.com/fleetdm/fleet/pull/11304 (still my previous comment applies)